shared_libraries/orca_shared/database/schemas/output.json

```diff
@@ -9,24 +9,24 @@
             "description": "Admin database name used for URL connection string. Typically postgres for PostgreSQL RDS",
             "type": "string"
         },
-        "admin_username": {
+        "admin_user": {
             "description": "Admin (super user) database user name used for URL connection string. Typically postgres for PostgreSQL RDS",
             "type": "string"
         },
-        "admin_password": {
+        "admin_user_password": {
             "description": "Admin database user password used for URL connection string.",
             "type": "string"
         },
-        "user_username": {
+        "app_user": {
             "description": "Application non-privelaged database user name used for URL connection string.",
             "type": "string"
         },
-        "user_password": {
+        "app_user_password": {
             "description": "Application user database password used for URL connection string.",
             "type": "string"
         },
-        "user_database": {
-            "description": "Application user database name used for URL connection string.",
+        "database": {
+            "description": "Application database name used for URL connection string.",
             "type": "string"
         },
         "host": {
```

Comment:
Shouldn't the variable names stay the same? `admin_username`, `admin_password`, `user_username`, `user_password` and `user_database`